Sources/MapboxCoreNavigation/EHorizon/DistancedRoadObject.swift

```diff
@@ -108,15 +108,15 @@ public enum DistancedRoadObject {
             let info = native.distanceInfo.getPolygonDistanceInfo()
             self = .polygon(identifier: native.roadObjectId,
                             type: RoadObjectType(native.type),
-                            distanceToNearestEntry: info.distanceToNearestEntry,
```

Comment:
I'm hesitant about this change, but to be more defensive I'd rather return an optional value instead of crashing or asserting.